ansible/roles/sufia/tasks/main.yml

```diff
@@ -92,6 +92,11 @@
   when: resque_installed.stat.exists
 
 - block:
+  - name: ensure the Noid minter statefile dir exists
+    file:
+      path: '{{ project_noid_statefile | dirname }}'
+      state: directory
+
```

Comment:
What do you think about adding a task to make sure the permissions and ownership of the minter state file are correct?  E.g.,

```
- name: ensure the Noid minter statefile has correct ownership and permissions
  file:
    path: '{{ project_noid_statefile }}'
    owner: '{{ project_user }}'
    mode: '0644'
    state: file
```

This wouldn't have any effect if the minter state file doesn't already exist, but it would serve to enforce correct permissions when it does, if those somehow get set incorrectly externally.  It also serves to make explicit what we expect the correct permissions to be for the minter state file.